templates/spec-s2i.tpl

```diff
@@ -30,13 +35,14 @@ stepTemplate:
   env:
 {{- $variables := list
       "params.IMAGE"
+      "params.IMAGE_SCRIPTS_URL"
       "params.SUBDIRECTORY"
       "params.STORAGE_DRIVER"
       "params.BUILD_EXTRA_ARGS"
       "params.PUSH_EXTRA_ARGS"
       "params.SKIP_PUSH"
       "params.TLS_VERIFY"
-      "params.VERBOSE"
+      "params.VERBOSE"         
```

Comment:
Make sure the editor is trimming extra white spaces at the end of each line, please.